File: tunnel-cli-common/src/main/java/com/tuuzed/tunnel/cli/common/LogLevel.java
package com.tuuzed.tunnel.cli.common;

import org.apache.log4j.Level;

public enum LogLevel {

    ALL(Level.ALL),
    TRACE(Level.TRACE),
    DEBUG(Level.DEBUG),
    INFO(Level.INFO),
    WARN(Level.WARN),
    ERROR(Level.ERROR),
    OFF(Level.OFF);

    public final Level level;


    LogLevel(Level level) {
        this.level = level;
    }
}

File: tunnel2-t2cli/src/main/kotlin/tunnel2/t2cli/LogLevel.kt
package tunnel2.t2cli

import org.apache.log4j.Level

enum class LogLevel(val level: Level) {

    ALL(Level.ALL),
    TRACE(Level.TRACE),
    DEBUG(Level.DEBUG),
    INFO(Level.INFO),
    WARN(Level.WARN),
    ERROR(Level.ERROR),
    OFF(Level.OFF)
}
